### PR TITLE
Adding Classifiers, Mappers, Incident Type, Incident Fields

### PR DESCRIPTION
**Pull request by alannix-lw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - Pending https://github.com/demisto/dockerfiles/pull/6318 for final container version bump.

## Description
This release will add Classifiers, Mappers, Incident Types and Incident Fields to make integrating with Lacework more seamless.  It also adds support for Lacework Organizations.

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Lacework/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+#### Classifiers
+##### New: Lacework - Classifier
+- Lacework Incident Classifier (Available from Cortex XSOAR 6.0.0).
+##### New: Lacework
+- 1.1.0 (Available from Cortex XSOAR 5.0.0).
```
Comment (barchen1) [inline Packs/Lacework/ReleaseNotes/1_1_0.md:6]: This line is for the 5_9_9 classifier

File: Packs/Lacework/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Incident Types
+- **Lacework Event **
+
+#### Integrations
+##### Lacework
+- 1.1.0
+- Added support for Lacework Organizations and Sub-Accounts
+- Updated the Docker image to: *demisto/lacework:1.0.0.23290*.
+
```
Comment (barchen1) [inline Packs/Lacework/ReleaseNotes/1_1_0.md:26]:
```suggestion
- Added support for Lacework Organizations and Sub-Accounts
- Updated the Docker image to: *demisto/lacework:1.0.0.23290*.

```

File: Packs/Lacework/ReleaseNotes/1_1_0.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#### Mappers
+##### New: Lacework - Incoming Mapper
+- 1.1.0 (Available from Cortex XSOAR 6.0.0).
```
Comment (barchen1) [inline Packs/Lacework/ReleaseNotes/1_1_0.md:29]: Please write a short line about the mapper instead.

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please add a description to all incident fields.

Let me know if you need help

File: Packs/Lacework/Classifiers/classifier-Lacework_5_9_9.json [hunk 4]
```diff
@@ -0,0 +1,78 @@
+{
+    "brandName": "Lacework",
+    "fromVersion": "5.0.0",
+    "toVersion": "5.9.9",
+    "custom": true,
+    "defaultIncidentType": "Lacework Event",
+    "id": "612fd5a2-31bf-4a83-8026-d4275ac5da55",
```
[the remark this answers is not in the document]
Comment (alannix-lw) [reply Packs/Lacework/Classifiers/classifier-Lacework_5_9_9.json]: Thank you! Should be resolve with this: https://github.com/demisto/content/pull/15014/commits/38f4fd0441a897a80a758b1510b7fb6afc289c61

File: Packs/Lacework/ReleaseNotes/1_1_0.md (hunk 1 above)
Comment (alannix-lw) [reply Packs/Lacework/ReleaseNotes/1_1_0.md]: Thanks again @barchen1!  I cleaned this up here: https://github.com/demisto/content/pull/15014/commits/0e730ad4e6fe6f17b48bff640fe144d830eb1fc2

The redundant versions numbers came from the `demisto-sdk update-release-notes` command - so wasn't sure if they were required or not.

File: Packs/Lacework/Classifiers/classifier-Lacework_5_9_9.json [hunk 5]
```diff
@@ -0,0 +1,78 @@
+{
+    "brandName": "Lacework",
+    "fromVersion": "5.0.0",
+    "toVersion": "5.9.9",
+    "custom": true,
+    "defaultIncidentType": "Lacework Event",
+    "id": "fcb9f8bc-4b35-4407-9329-e08f17d8d711",
```
Comment (barchen1) [inline Packs/Lacework/Classifiers/classifier-Lacework_5_9_9.json:7]: Please change the id

**Review by barchen1 — CHANGES_REQUESTED** [review]

**Pull request merged**